packages/create-cloudflare/src/cli.ts

```diff
@@ -138,13 +138,13 @@ const configure = async (ctx: C3Context) => {
 	await installWrangler();
 	await installWorkersTypes(ctx);
 
-	await updateWranglerToml(ctx);
-
 	const { template } = ctx;
 	if (template.configure) {
 		await template.configure({ ...ctx });
 	}
 
+	await updateWranglerToml(ctx);
```

Comment:
@jculvey I'm moving this because we're creating the toml file in the `configure` phase, what do you think?

Alternatively if it's better I can put the toml file creation into `generate` and leave this as is

what do you prefer?

Reply:
I don't think we can move it, because the `pre-existing` template needs to run wrangler commands to download the existing script file in the configure phase, and this will fail if the name is still `<TBD>`. Can you try running this with `--existing-script` and see if it fails?

`copyFiles` can normally happen because it happens in the generation step. I guess we have to manually copy over the wrangler file because of the `destinationDir` being either `.` or `src/`. I think moving this code to `generate` is the better option.

Reply:
Now that I think about it, we should definitely leave a comment here on why this function call needs to be in this particular place. Silly me for not leaving one last time 😓

Reply:
Don't we have any tests that would fail? 😓 

I guess we don't have e2es for existing scripts uh?